syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.examples.cabonlinechallenge";
option java_outer_classname = "CabonlineChallengeProto";
option objc_class_prefix = "HLW";

package cabonlinechallenge;

service CabonlineChallenge {
  rpc SetChallengeData (SetChallengeDataRequest) returns (SetChallengeDataReply) {}
}

message SetChallengeDataRequest {
        string requestcontent = 1;
}

message SetChallengeDataReply {
        string responsecontent = 1;
}

